packages/web-react/src/components/CheckboxField/README.md

```diff
@@ -1,11 +1,18 @@
 # CheckboxField
 
 CheckboxField enables the user to check/uncheck choice. It has input, a label,
-and an optional message. It could be disabled or have a validation state. The label could be hidden
+and an optional validationText. It could be disabled or have a validation state. The label could be hidden
```

Comment:
this should also be a helper text or something. Definitely not like this. because validation state is not written here as validationState.

Reply:
Ok, fixed.